handbook/engineering/distribution/index.md

```diff
@@ -66,7 +66,7 @@ Distribution team members may also be involved in other areas of Sourcegraph not
     - Making the debugging process for common problems seamless and straightforward
     - Making reporting issues with all needed information easy
     - Ensuring logs/tracing are not overly verbose, identify most useful information for solving problems
-    - **Primary owners**: @dax (@beyang and @stephen in the interim)
+    - **Primary owners**: @stephen
```

Comment:
Are you actually the owner of this or should we either (1) leave this as unowned, or (2) delete this section entirely for now if it is not a priority?

Should Dax be removed from line 49 single-container installation & upgrade experience?

Reply:
There is no better owner for now, so I am the de-facto owner. I am the owner because: it is on me to assess what here is or is not a priority and find an appropriate owner.

Line 49 -> Yes, updated.